lib/configfiles/leap_42.1.xml

```diff
@@ -0,0 +1,1474 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<froxlor>
+	<distribution name="openSUSE" codename="Leap" version="42.1" defaulteditor="/usr/bin/vim">
+		<services>
+			<!-- HTTP -->
+			<service type="http" title="{{lng.admin.configfiles.http}}">
+				<!-- general HTTP commands -->
+				<general>
+					<commands index="1">
+						<command>
+							<visibility mode="isdir">{{settings.system.apacheconf_vhost}}
+							</visibility>
+							<content><![CDATA[mkdir -p {{settings.system.apacheconf_vhost}}]]></content>
+						</command>
+						<command>
+							<visibility mode="isfile">{{settings.system.apacheconf_vhost}}
+							</visibility>
+							<content><![CDATA[touch {{settings.system.apacheconf_vhost}}]]></content>
+						</command>
+						<command><![CDATA[chown root:0 {{settings.system.apacheconf_vhost}}]]></command>
+						<command><![CDATA[chmod 0600 {{settings.system.apacheconf_vhost}}]]></command>
+						<command>
+							<visibility mode="isdir">{{settings.system.apacheconf_diroptions}}
+							</visibility>
+							<content><![CDATA[mkdir -p {{settings.system.apacheconf_diroptions}}]]></content>
+						</command>
+						<command>
+							<visibility mode="isfile">{{settings.system.apacheconf_diroptions}}
+							</visibility>
+							<content><![CDATA[touch {{settings.system.apacheconf_diroptions}}]]></content>
+						</command>
+						<command><![CDATA[chown root:0 {{settings.system.apacheconf_diroptions}}]]></command>
+						<command><![CDATA[chmod 0600 {{settings.system.apacheconf_diroptions}}]]></command>
+						<command><![CDATA[mkdir -p {{settings.system.documentroot_prefix}}]]></command>
+						<command><![CDATA[mkdir -p {{settings.system.logfiles_directory}}]]></command>
+						<command><![CDATA[mkdir -p {{settings.system.mod_fcgid_tmpdir}}]]></command>
+						<command><![CDATA[chmod 1777 {{settings.system.mod_fcgid_tmpdir}}]]></command>
+						<command>
+							<visibility mode="notempty">{{settings.system.deactivateddocroot}}
+							</visibility>
+							<content><![CDATA['mkdir -p {{settings.system.deactivateddocroot}}]]></content>
+						</command>
+						<command><![CDATA[a2dismod userdir]]></command>
+					</commands>
+				</general>
+				<!-- HTTP Apache -->
+				<daemon name="apache" version="2.4" title="Apache 2.4">
```

Comment:
sorry I do not understadn waht to do?
Please help.
Were should be default=true  added?